IBPSA/ThermalZones/ReducedOrder/RC/OneElement.mo

```diff
@@ -129,8 +139,25 @@ model OneElement "Thermal Zone with one element for exterior walls"
     final C_start=C_start,
     final C_nominal=C_nominal,
     final mSenFac=mSenFac,
-    final use_C_flow=false) if VAir > 0 "Indoor air volume"
-    annotation (Placement(transformation(extent={{38,-10},{18,10}})));
+    final use_C_flow=false) if VAir > 0 and not use_moisture_balance
+    "Indoor air volume"
+    annotation (Placement(transformation(extent={{42,-26},{22,-6}})));
+  Fluid.MixingVolumes.MixingVolumeMoistAir volMoiAir(
```

Comment:
This makes me wonder why we have a separate model for `MixingVolume` with and without moisture..?

Reply:
I wonder too.